my-app1/src/Display.js

```diff
@@ -0,0 +1,26 @@
+import React ,{ Component } from 'react';
+
+class Display extends Component {
+	constructor(props) {
+		super(props);
+		this.state = {
+			arr: this.props.arr
+		};
+	}
+	static getDerivedStateFromProps(newProps, prevState) {
```

Comment:
also before the definition of a new function, It's recommended that you should leave one line before /after.